package.json

```diff
@@ -38,17 +38,18 @@
     "interval"
   ],
   "devDependencies": {
-    "@storybook/addon-options": "4.2.0-alpha.8",
-    "@storybook/addons": "4.2.0-alpha.8",
-    "@storybook/react": "4.2.0-alpha.8",
+    "babel-loader": "^8.0.6",
```

Comment:
what's up with `babel`?

Reply:
It's required for `storybook` v5. Without it, `storybook` won't build/load. They added something to their docs about it becoming a dependency.